build.ps1

```diff
@@ -26,6 +26,16 @@ function ShouldNPMInstall {
 
 $purge = $args | Where-Object { $_ -like "--purge" }
 $pack = $args | Where-Object { $_ -like "--pack" }
+$pack_build = $args | Where-Object { $_.startsWith("--pack.build=") }
+if ($pack_build) {
+    # Example: --pack.build=0.1.$(Build.BuildNumber)/$(Build.SourceVersion)
+    Write-Host ($pack_build)
+    $tokens = $pack_build.split("=")[1].trim().split("/")  
+    $pack_build = $pack_build = $tokens[0]
+} else {
+    $pack_build = "0.1.0" # default value

```

Comment:
I am thinking we don't want to give a default version here just in case everyone who missed this parameter would  push to the same spot. What happens if --version is missing to nuget push? We should match that behavior

Reply:
@shhsu 
Without the -version, it fails with the below error. 

```
C:\gitroot\IIS.WebManager\src>nuget pack . -OutputDirectory ..\dist
Attempting to build package from 'wac-iis.nuspec'.
Value cannot be null or an empty string.
Parameter name: value
```

How about setting 0.0.0.0 for the default value, then?